### PR TITLE
Manual

### PR DESCRIPTION
**Pull request by mem48**

### CODE REVIEW

File: js/ga_optout.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function setCookie(cvalue) {
   var d = new Date();
   d.setTime(d.getTime() + (1000 * 24 * 60 * 60 * 1000));
   var expires = "expires="+d.toUTCString();
-  document.cookie = "NTPtrack=" + cvalue + ";" + expires + ";path=/";
```
Comment (Robinlovelace) [inline js/ga_optout.js:38]: Good typo fix.

File: index.html [hunk 2]
```diff
@@ -270,7 +273,7 @@ <h3>Welcome to NPT Scotland: v0.1 Fast Track</h3>
   <p>
     <select id="data_zone_input" name="data_zone_input" onchange="switch_data_zones()" aria-label="Data zone layer">
       <option value="population_density" selected>Population density (per hectare)</option>
-      <option value="SIMD2020v2_Decile">Index of Muliple Deprivation (2020)</option>
```
Comment (Robinlovelace) [inline index.html:273]: :+1:

**Review by Robinlovelace — COMMENTED** [review]
Good to have a manual. The formatting is weird with a huge amount of empty space, surely we can use some styling framework to improve this? Suggestion: ask the community, after this v0.1 Launch.

I plan to make some tweaks to the IP section. Then we're good to go.

**Comment by Robinlovelace** [comment]
White space shown below:

image

**Comment by mem48** [comment]
E.g.
image

BBC new use 800px for the content, although they have the side bar that adds a little addtional width.

**Comment by Robinlovelace** [comment]
> Its pretty standard to cap the max width of a page at 800px. You must be on a very wide screen. If the text filled the whole width it would look weird and be hard to read.

Makes sense. Maybe worth centering the text?

**Comment by Robinlovelace** [comment]
@mem48 if you could sanity check my latest commit that splits IP and Open Source sections in 2, following advice from @mvl22, that would be appreciated.

**Comment by Robinlovelace** [comment]
Also, I plan to format the about page with the Prettify VS Code extension...

**Comment by mem48** [comment]
Looks good, but there is some overlap with https://nptscot.github.io/data/ which is based on https://www.carbon.place/data/ do we want to consolidate to one place?

**Comment by mem48** [comment]
Key difference is that it make reference to separate licences for code and data. GNU Affero General Public License v3.0 is not a data licence an so does not hold up legally.

**Comment by mem48** [comment]
For PBCC I went for a slightly more restrictive licencing on the code than the data. The GNU Affero General Public License v3.0 is share and share alike, while [Open Data Commons Attribution License](https://opendatacommons.org/licenses/by/). is pretty much do what you want. WHich I think makes sence to distingish between the tool and its outputs.

**Comment by Robinlovelace** [comment]
Can the About and Data sections all have the same nice formatting as the Manual now?

**Comment by Robinlovelace** [comment]
> For PBCC I went for a slightly more restrictive licencing on the code than the data. The GNU Affero General Public License v3.0 is share and share alike, while [Open Data Commons Attribution License](https://opendatacommons.org/licenses/by/). is pretty much do what you want. WHich I think makes sence to distingish between the tool and its outputs.

Sure but we don't have any data currently. Can cross that bridge...

**Comment by Robinlovelace** [comment]
Looking great, just the privacy section could do with being brought in line with central formatting. Overall: good to go :rocket: 

Peek 2023-03-31 22-50

**Review by Robinlovelace — APPROVED** [review]

**Comment by mvl22** [comment]
Yes, max-width 800px is good practice as it constrains the lines to avoid long lines being very hard to read and scan.

**Pull request merged**